athina_client/services/athina_api_service.py

```
@@ -645,3 +645,80 @@ def update_dataset_cells(dataset_id: str, cells: List[Dict[str, Any]]):
         except Exception as e:
             raise CustomException("Error updating dataset cells", str(e))
 
+    @staticmethod
+    @retry(stop_max_attempt_number=2, wait_fixed=1000)
+    def create_eval_run(create_eval_run: bool, eval_configs: List[Dict[str, Any]], dataset_id: Optional[str] = None):
```

Ruff (0.8.2)

Undefined name `Optional`

(F821)